designsafe/static/scripts/data-depot/components/projects/pipeline-authors/pipeline-authors.component.js

```diff
@@ -53,6 +55,15 @@ class PipelineAuthorsCtrl {
                 }
             };
             this.verifyAuthors(this.experiment.value.authors);    
+            if (this.projType === 'experimental') {
+                this.placeholder = 'Experiment';
+            } else if (this.projType === 'simulation') {
+                this.placeholder = 'Simulation';
+            } else if (this.projType === 'hybrid_simulation') {
+                this.placeholder = 'Hybrid Simulation';
+            } else if (this.projType === 'field_recon') {
```

Comment:
What about `projType == 'other'`?

Reply:
`projType == 'other'` projects handle authors differently than other projects. We use `pipeline-team.component.js` in this case.